pyi.py

```diff
@@ -34,6 +34,8 @@ def unparse(node: ast.AST) -> str:
 
 
 if TYPE_CHECKING:
+    from _ast import _Slice
```

Comment:
We probably shouldn't be importing from the private `_ast` module.

Reply:
It's a typeshed type alias that doesn't exist in typeshed's stub for `ast`. I guess we could duplicate the alias in our code here. Or just add a comment for why we're importing this weird private thing -- what do you prefer?

Reply:
Or I guess we could just revert the addition of the explicit annotation on line 669

Reply:
We shouldn't use private aliases from typeshed because the people who maintain typeshed (not sure who does that) can change them at any time.

Agree that if we're not actually running pyright we can omit the explicit annotations for those lists.